docs/conf.py

```diff
@@ -63,10 +60,34 @@
 #
 html_theme = "pydata_sphinx_theme"
 
+html_theme_options = {
+    "use_edit_page_button": True,
+    "icon_links": [
+        {
+            "name": "GitHub",
+            "url": "https://github.com/ReproNim/neurodocker",
+            "icon": "fa-brands fa-github",
+        },
+        {
+            "name": "Docker Hub",
+            "url": "https://hub.docker.com/r/repronim/neurodocker",
+            "icon": "fa-brands fa-docker",
+        },
+    ],
+}
+
+
+html_context = {
+    "github_user": "ReproNim",
+    "github_repo": "neurodocker",
+    "github_version": "master",
+    "doc_path": "docs",
+}
+
```

Comment:
Those are just to add links to the repos on github or docker hub in the navbar and to add an "edit source" link when browsing the doc.

Reply:
thanks for adding this. i always appreciate when i'm able to easily browse the source from the documentation.